webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux.js';
```

Comment:
In node, the file extension is mandatory in imports. With webpack the `.js` extension is optional. I believe there are also lint rules set up that will complain if you use the extension in a React app. So throughout this file, remove the `.js` extensions.